_data/simple-icons.json

```
@@ -16860,6 +16860,11 @@
 		"hex": "3544B1",
 		"source": "https://umbraco.com"
 	},
+	{
+		"title": "Umbrel",
+		"hex": "5351F3",
+		"source": "https://github.com/getumbrel/umbrel/blob/bd8ced1b193d6d0eaa4a6811ba5a8c0cbb6776f2/packages/ui/public/figma-exports/umbrel-app.svg"
```

```suggestion
		"source": "https://umbrel.com"
```

Perhaps it's better to change the source to our main website? Since the logo's always present on it.